src/plexil-adapter/OwInterface.cpp

```diff
@@ -833,11 +838,6 @@ double OwInterface::getTiltVelocity () const
   return JointTelemetryMap[Joint::antenna_tilt].velocity;
 }
 
-bool OwInterface::imageReceived () const
-{
-  return ImageReceived;
-}
-
```

Comment:
The method has been removed, but the declaration is still present in OwInterface.h

Reply:
OwInterface.h:67

Reply:
Forgot to remove - gone now.